### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by neoneye** [rubenv/sql-migrate#108]
I miss an easy tutorial that describes in greater detail how to use `sql-migrate`. I searched google without luck.

The [test-migrations](https://github.com/rubenv/sql-migrate/tree/master/test-migrations) dir currently contains 2 files. It would be nice to see a longer list of migrations. Perhaps 7 migrations, showing more sql commands. This would be a great way to demo how to use the `-limit=x` parameter.

**Comment by opensas** [comment]
I agree, it's such a simple, yet powerful tool that it's a shame it doesn't have more examples and documentation
